.env.dist

```diff
@@ -16,8 +17,8 @@ PRODUCT_RETAILER_ID='YOUR_PRODUCT_RETAILER_ID'
 VIBER_SERVICE_MESSAGE_ID='YOUR_VIBER_SERVICE_MESSAGE_ID'
 
 # Signing messages
-VONAGE_SIGNATURE_SECRET = 'YOUR_VONAGE_SIGNATURE_SECRET'
-VONAGE_SIGNATURE_SECRET_METHOD = 'METHOD_USED_TO_SIGN_YOUR_SECRET'
+VONAGE_SIGNATURE_SECRET=
```

Comment:
can we keep the values?

Reply:
Due to how the SDK has always worked, keeping these in causes the non-signed SMS samples to fail, so it's best to keep these blank.